passphrase/passphrase.go

```diff
@@ -143,9 +147,14 @@ func (br *boundRetriever) requestPassphrase(keyName, alias string, createNew boo
 }
 
 func (br *boundRetriever) verifyAndConfirmPassword(stdin *bufio.Reader, retPass, displayAlias, withID string) error {
-	if len(retPass) < 8 {
-		fmt.Fprintln(br.out, "Passphrase is too short. Please use a password manager to generate and store a good random passphrase.")
-		return ErrTooShort
+	pass, err := br.validateFunc(retPass)
+	if err != nil {
+		fmt.Fprintln(br.out, "Passphrase is invalid: %s", err)
+		return err
+	}
+
+	if strings.TrimSpace(pass) == "" {
```

Comment:
Should we always perform a TrimSpace, or allow custom validators to accept whitespace?

Also; should the default validator always be performed (as an enforced minimum validation, even if a custom validator is used)? Wondering if there would be valid reasons to have a custom validator that accepts passphrases < 8 characters